index.js

```diff
@@ -1,10 +1,10 @@
 const { Client, GatewayIntentBits } = require('discord.js');
-const client = new Client({ 
-  intents: [ 
+const client = new Client({
+  intents: [
     GatewayIntentBits.Guilds,
-    GatewayIntentBits.GuildMembers, 
-    GatewayIntentBits.MessageContent 
-  ] 
+    GatewayIntentBits.GuildMembers,
+    GatewayIntentBits.MessageContent
+  ]
```

Comment:
Whoops, looks like my editor stripped some whitespace.  This shouldn't be part of this change.

Reply:
Got it! No worries, I'll remove that extra line and make sure everything is properly formatted. Thanks for catching that!